app/_src/quickstart/docker-demo.md

```diff
@@ -0,0 +1,716 @@
+
+   Create a token for the `kv` data plane proxy to authenticate with the control plane:
+
+   ```sh
+   kumactl generate dataplane-token \
+     --tag kuma.io/service=kv \
+     --valid-for 720h \
+     > {{ tmp }}/token-kv
+   ```
+
+2. **Start the container**
+
+   ```sh
+   docker run \
+     --detach \
+     --name {{ kuma-demo }}-kv \
+     --hostname kv \
+     --network {{ kuma-demo }} \
+     --ip {{ ip_abc }}.2 \
+     --volume {{ tmp }}:/demo \
+     ghcr.io/kumahq/kuma-counter-demo:debian-slim
+   ```
+
+3. **Prepare the container**
+
+   Enter the container to run further commands.
+
+   {% if version != "preview" %}
+   ```sh
+   docker exec --tty --interactive --privileged {{ kuma-demo }}-kv bash
+   ```
+   {% else %}
+   ```sh
+   docker exec \
+     --tty \
+     --interactive \
+     --privileged \
+     --env {{ KUMA_PREVIEW_VERSION }} \
+     {{ kuma-demo }}-kv bash
+   ```
+   {% endif %}
+
+   {{ warning-run-inside-container | indent }}
+
+   1. **Install tools and create data plane proxy user**
+
+      Install the required tools for downloading {{ Kuma }} binaries, setting up the [transparent proxy]({{ docs }}/production/dp-config/transparent-proxying/), and create a dedicated user for the data plane proxy:
+
+      - `curl`: Needed to download the {{ Kuma }} binaries.
+      - `iptables`: Required to configure the transparent proxy.
+
+      Run the following commands:
+
+      {{ code-block-install-tools-create-user | indent | indent }}
+
+   2. **Set the zone name**
+
+      Give the `kv` instance a name. The demo application will use this name to show which `kv` instance is being accessed.
+
+      ```sh
+      curl localhost:5050/api/key-value/zone \
+        --header 'Content-Type: application/json' \
+        --data '{"value":"local-demo-zone"}'
+      ```
+
+   3. **Start the data plane proxy**
+
+      ```sh
+      runuser --user {{ kuma-data-plane-proxy }} -- \
+        /usr/local/bin/kuma-dp run \
+        --cp-address https://control-plane:5678 \
+        --dataplane-token-file /demo/token-kv \
+        --dataplane-file /demo/dataplane.yaml \
+        --dataplane-var name=kv \
+        --dataplane-var address={{ ip_abc }}.2 \
+        --dataplane-var port=5050 \
+        > /demo/logs-data-plane-proxy-kv.log 2>&1 &
+      ```
+
+   4. **Install the transparent proxy**
+
+      {{ warning-tproxy-command-run-inside-container | indent | indent }}
+
+      ```sh
+      kumactl install transparent-proxy \
+        --config-file /demo/config-transparent-proxy.yaml \
+        > /demo/logs-transparent-proxy-install-kv.log 2>&1
+      ```
+
+   5. **Exit the container**
+
+      Key/Value Store is now set up and running. You can safely exit the container as the configuration is complete:
+
+      ```sh
+      exit
+      ```
+
+4. **Check if service is running**
+
+   To confirm the service is set up correctly and running, use the {{ kumactl }} to inspect the services:
+
+   ```sh
+   kumactl inspect services
```

Comment:
```suggestion
   kumactl get meshservices
```

I think we should not use inspect services anymore as we are using meshservices in this guide

Reply:
addressed